.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/
 vendor/
 hey-apm
+build
+Gopkg.lock
```

Comment:
what is the reason you added this? i don't think the lock file should be ignored, is there to make sure we have consistent versions of dependencies.

Reply:
ok, I removed it.